### PR TITLE
full deprecate scheduler.lr setting

### PR DESCRIPTION
**Pull request by sfc-gh-mwyatt**
Since launch we had `optimizer.learning_rate` and `scheduler.learning_rate`. Fully deprecating `scheduler.learning_rate` and adding an error message if a user attempts to set this config field.

### CODE REVIEW

**Review by sfc-gh-sbekman — APPROVED** [review]
Thank you for fixing, Mike.
